### ISSUE TITLE
Middle mouse click outside of terminal

### ISSUE THREAD

**Issue by 2manyvcos** [platformio/platformio-atom-ide-terminal#264]
# Environment
Linux Mint 18.1 Cinnamon
```
$> atom --version
Atom    : 1.18.0
Electron: 1.3.15
Chrome  : 52.0.2743.82
Node    : 6.5.0
```

I should also note that middle mouse button is used for pasting in Linux Mint by default.

# Reproducing
 - Have some editor tabs open
 - Have some text in clipboard
 - Click into the console so that it's got focus
 - Click on an editor tab with middle mouse button to close it

# Expected behavior
 - the tab get's closed
 - the console doesn't get influenced

# Actual behavior
 - the tab get's closed
 - the text in the clipboard get's inserted into the focused console pane

**Comment by 2manyvcos** [comment]
Sorry, this is an [atom issue](https://github.com/atom/atom/pull/15183).

**Comment by hatsuo** [comment]
This is fixed now in Atom Version 1.21.2.
